credentials/credentials.go

```diff
@@ -178,8 +178,14 @@ type TransportCredentials interface {
 //
 // This API is experimental.
 type Bundle interface {
+	// TransportCredentials returns the transport credentials from the Bundle.
+	//
+	// Implementations must return non-nil transport credentials.
 	TransportCredentials() TransportCredentials
+
+	// PerRPCCredentials returns the per-RPC credentials from the Bundle.
```

Comment:
"May be nil if per-RPC credentials are not needed"?

Reply:
Done.